.github/workflows/openssl.yml

```diff
@@ -0,0 +1,66 @@
+# Test old OpenSSL version(s)
+#
+# As we are using `ubuntu-latest`, we cannot rely on the installed
+# `openssl` package to test all supported versions.
+# Instead, this workflow installs an explicit version, builds it,
+# and test the tls package with it.
+name: OpenSSL
+
+on: [push, pull_request]
+
+jobs:
+  main:
+    name: Run
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ ubuntu-latest ]
+        dc:
+          - dmd-latest
+#          - ldc-latest
+        # Currently only testing v1.0.x as 1.1.0 is tested via ubuntu-20.04
+        openssl_version:
+          - 1.0.2u
+
+    runs-on: ${{ matrix.os }}
+    timeout-minutes: 60
+
+    steps:
+    - uses: actions/checkout@v2
```

Comment:
I have many projects to upgrade then :)

Reply:
I use `dependabot` to alert me, which is now a Gtihub official thing, especially when some CVE is reported on the attached repository. The caching action is a more practical change tho, since it may have some kind of performance boost, e.g. different caching mechanism, although I can't confirm that statement.